editor/src/components/navigator/navigator-item/component-picker.tsx

```diff
@@ -234,48 +324,58 @@ function iconPropsForIcon(icon: Icon): IcnProps {
 interface ComponentPickerOptionProps {
   component: InsertMenuItemGroup
   onItemClick: (elementToInsert: InsertableComponent) => React.MouseEventHandler
+  onItemHover: (elementToInsert: InsertMenuItemValue) => React.MouseEventHandler
+  currentlySelectedKey: string | null
 }
 
 const ComponentPickerOption = React.memo((props: ComponentPickerOptionProps) => {
-  const { component, onItemClick } = props
+  const { component, onItemClick, onItemHover, currentlySelectedKey } = props
 
   const variants = component.options
 
   const name = component.label
 
   return (
     <div>
-      {variants.map((v) => (
-        <FlexRow
-          key={`${name}-${v.label}`}
-          css={{
-            marginLeft: 8,
-            marginRight: 8,
-            borderRadius: 4,
-            // indentation!
-            paddingLeft: 8,
-            color: '#EEE',
-            '&:hover': {
-              background: '#007aff',
-              color: 'white',
-            },
-          }}
-          onClick={onItemClick(v.value)}
-        >
-          <UIGridRow
-            variant='|--32px--|<--------auto-------->'
-            padded={false}
-            // required to overwrite minHeight on the bloody thing
-            style={{ minHeight: 29 }}
+      {variants.map((v) => {
+        const selectedStyle =
+          v.value.key === currentlySelectedKey
+            ? {
+                background: '#007aff',
+                color: 'white',
+              }
+            : {}
+        return (
+          <FlexRow
+            key={`${name}-${v.label}`}
             css={{
```

Comment:
Now that we're not using `&:hover`, we no longer need to use the `css` key here

Reply:
Changed it, interesting the `FlexRow` still requires an empty `css` prop